CalibCalorimetry/HcalTPGAlgos/src/HcaluLUTTPGCoder.cc

```diff
@@ -544,6 +544,11 @@ void HcaluLUTTPGCoder::update(const HcalDbService& conditions) {
             else if (adc >= mipMax)
               lut[adc] |= QIE11_LUT_MSB1;
           }
+
+	  //Zeroing the 4th depth in the trigger towers where |ieta| = 16 to match the behavior in the uHTR firmware, where the 4th depth is not included in the sum over depths when constructing the TP energy for this tower.
+	  if (abs(cell.ieta()) == 16 && cell.depth() == 4) {
```

Comment:
I think that you want to do this only if `topo_->triggerMode() >= HcalTopologyMode::TriggerMode_2021` so that this channel is only zeroed out in the trigger tower sums for Run 3. The `depth=4` contribution was included in the trigger tower sums for |ieta|=16 during 2017 and 2018.

Reply:
Thanks for catching this Chris! I modified the if statement as you suggested so that this is only applied for Run3.